web/webkit/src/main/scala/net/liftweb/http/LiftResponse.scala

```diff
@@ -104,10 +104,17 @@ case class ResetContentResponse() extends LiftResponse with HeaderDefaults {
  * 400 Bad Request
  *
  * Your Request was missing an important element. Use this as a last resort if
- * the request appears incorrect.
+ * the request appears incorrect. Use the `message` to indicate what was wrong
+ * with the request, if that does not leak important information.
  */
-case class BadResponse() extends LiftResponse with HeaderDefaults {
-  def toResponse = InMemoryResponse(Array(), headers, cookies, 400)
+case class BadRequest(message: String = "") extends LiftResponse with HeaderDefaults {
```

Comment:
Shouldn't this really be `BadRequestResponse`? (cf. `OkResponse`, `RedirectResponse`, `JsonResponse`, `CreatedResponse`)